arduino/tm1638-gauge.ino

```diff
@@ -116,11 +116,11 @@ void loop() {
      char* neutral = "n";                // sets the character for neutral
      char* reverse = "r";                // sets the character for reverse
 
-     if (gear >= 1 and gear <10 ){
+     if (gear >= -1 and gear <10 ){
```

Comment:
I think you don't want this change. It'll will keep both the neutral check below, as well as the explicit check for `gear == -1` from being used